src/runtime_src/doc/toc/xbflash2.rst

```diff
@@ -11,9 +11,15 @@ The Xilinx (R) Board Flash utility (xbflash2) is a standalone command line utili
 
 In 2022.1 release, this utility is Early Access with limited validation.
 
-This tool supported for all Alveo platforms, U2 and U30 platforms.
+This tool supported for all Alveo platforms.
 
-This tool doesn't come with XRT package, it comes as separate xbflash package.
+This tool doesn't require XRT Package and it doesn't come with XRT package, it comes as separate xbflash package.
+
+``xbflash2`` tool is available in the Alveo card web page, getting started session, xbflash2 tab.
+
+For example: https://www.xilinx.com/products/boards-and-kits/alveo/u50.html#xbflash2
+
+After xbflash package installation, content goes to ``/usr/local/bin``
```

Comment:
I am still convinced this is not the best choice to keep in sync with the whole `/opt/xilinx` organization by default.
Further more the content is weird:
```
rkeryell@rk-xsj:~ (master)$ dpkg -L xrt-xbflash2
/opt
/opt/xilinx
/usr
/usr/local
/usr/local/bin
/usr/local/bin/xbflash2
```
At the end it makes more sense to have it in `/opt/xilinx/xrt/bin` like everything.

Reply:
Noted.